lightly_studio/src/lightly_studio/models/video.py

```diff
@@ -84,7 +84,8 @@ class VideoFrameBase(SQLModel):
     """The frame number of the video frame."""
     frame_number: int
 
-    """The timestamp of the video frame in seconds."""
+    """Presentation timestamp in the video's time_base and in seconds."""
+    frame_timestamp_pts: int
```

Comment:
Sidenote: "pts" means "presentation timestamp" so the variable name is "frame timestamp presentation timestamp". I don't mind though.